enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/ServiceDiscoveryEnricher.java

```diff
@@ -27,27 +28,33 @@
 import org.w3c.dom.Document;
 import org.w3c.dom.Node;
 
+import io.fabric8.kubernetes.api.builder.TypedVisitor;
+import io.fabric8.kubernetes.api.model.KubernetesListBuilder;
+import io.fabric8.kubernetes.api.model.ServiceBuilder;
+import io.fabric8.kubernetes.api.model.ServicePort;
 import io.fabric8.maven.core.util.Configs;
-import io.fabric8.maven.core.util.MapUtil;
 import io.fabric8.maven.enricher.api.BaseEnricher;
 import io.fabric8.maven.enricher.api.EnricherContext;
 import io.fabric8.maven.enricher.api.Kind;
 
 public class ServiceDiscoveryEnricher extends BaseEnricher {
     static final String ENRICHER_NAME        = "f8-service-discovery";
+    private boolean tried = false;
 
     //Default Prefix
-    static final String PREFIX    = "discovery.3scale.net/";
+    static final String PREFIX    = "discovery.3scale.net";
     //Service Annotations
     static final String DISCOVERY_VERSION = "discovery-version";
     static final String SCHEME            = "scheme";
     static final String PATH              = "path";
+    static final String PORT              = "port";
     static final String DESCRIPTION_PATH  = "description-path";
     //Service Labels
     static final String DISCOVERABLE      = "discoverable";
 
     private File springConfigDir;
-    private String path             = null;
+    private String path             = "/";
+    private String port             = "80";
```

Comment:
I would leave this null, also to decide whether auto-detection is required or not.

Reply:
Y I debated this, but @chirino seems to expect getting a value there regardless.

Reply:
I spoke with @chirino and changed it back